lib/hammer_cli_katello/content_host.rb

```diff
@@ -113,6 +113,46 @@ class TasksCommand < HammerCLIKatello::ListCommand
       build_options
     end
 
+    class AvailableIncrementalUpdates < HammerCLIKatello::ListCommand
+      resource :systems_bulk_actions, :available_incremental_updates
+      command_name 'needed-incremental-updates'
```

Comment:
is this for displaying needed incremental updates? could you somehow incorporate a verb into this command name?